contracts/LockedNORI.sol

```diff
@@ -263,7 +243,20 @@ contract LockedNORI is ERC777PresetPausablePermissioned {
   }
 
   /**
-   * @notice Batch version of `createGrant` with permit support.
+   * @notice Sets up a batch of vesting + lockup schedules and wraps the corresponding BridgedPolygonNORI
+   * using EIP712 permit access.
+   *
+   * ##### Requirements:
+   *
+   * - Can only be used when the contract is not paused.
+   * - Can only be used when the caller has the `TOKEN_GRANTER_ROLE` role.
+   *
+   * @param amounts The amount of each grant
+   * @param grantParams An array of `CreateTokenGrantParams` structs
+   * @param deadline The permit deadline
+   * @param v The permit v parameter
+   * @param r The permit r parameter
+   * @param s The permit s parameter
```

Comment:
```suggestion
   * @param grantParams An array of `CreateTokenGrantParams` structs
   * @param deadline The permit deadline.
   * @param v The permit v parameter.
   * @param r The permit r parameter.
   * @param s The permit s parameter.
```

These could honestly use better descriptions too, but considering the consumers of this contract, it's probably not worth the effort.